src/cache.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const QuickLRU = require('quick-lru');
+
+let lru = new QuickLRU({ maxSize: 1000 });
+
+/**
+ * Returns a memoized version of the function `fn`.
+ * @param {function} fn a function to memoize
+ * @param {function} hash a hash function to build a cache key.
```

Comment:
better to use a dedicated. `options` argument.
- easier to document
- easier to create TS decls
- easier to understand.
- easier for IDEs to typecheck

Reply:
The `cache` function is already using an `options` argument as the second argument. Nesting the function in the options argument, too doesn't make much sense.